cpio

```diff
@@ -0,0 +1,6 @@
+# To create a cpio archive
+ls | cpio -o > <archive.cpio>
```

Comment:
```suggestion
# Create a specific cpio archive:
ls | cpio --create > <path/to/archive.cpio>
```

- Optional change: `to` is not mandatory.
- Optional change: `a specific` may sound better.
- Optional change: maybe it's better to use a trailing comma like in TlDr pages.
- Optional change: it's better to use long options or present them alongside short ones. Long options can be faster memorized.
- Optional change: it's better to use `<path/to/file>` path placeholders.

Reply:
Hello, thank you for reviewing this
God Bless You
I will make sure to use these tips in my next contributions